pyproject.toml

```diff
@@ -56,6 +57,7 @@ lambda = [
 
 [dependency-groups]
 dev = [
+    "dask>=2025.9.1",
```

Comment:
I think this was beneficial for generating the test fixtures, but happy to remove it if you think that I should.